src/main/java/org/folio/circulation/resources/AbstractCirculationRulesEngineResource.java

```diff
@@ -9,8 +9,20 @@
 import java.util.Map;
 import java.util.concurrent.CompletableFuture;
 
+import io.vertx.core.Handler;
```

Comment:
This ordering appears to be different to the [provisionally agreed standard for import statements](https://issues.folio.org/browse/FOLIO-1667), please can these be changed (and all other files checked for similar issues).

Reply:
fixed